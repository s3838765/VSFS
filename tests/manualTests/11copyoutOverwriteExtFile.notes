NOTES V1.0
@overwriteFile
 This is the first line of an overwritten file.
 This is the second.
 This is the third and final line!
