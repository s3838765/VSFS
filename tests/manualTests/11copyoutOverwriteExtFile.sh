#! /usr/bin/sh
cp 11copyoutOverwriteExtFile.txt 11copyoutOverwriteExtFileTemp.txt
../../VSFS copyout 11copyoutOverwriteExtFile.notes overwriteFile 11copyoutOverwriteExtFileTemp.txt
cp 11copyoutOverwriteExtFileTemp.txt 11copyoutOverwriteExtFile.out
rm 11copyoutOverwriteExtFileTemp.txt
