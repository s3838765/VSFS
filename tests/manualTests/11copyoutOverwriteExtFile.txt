THIS IS AN EXTERNAL FILE!
THIS WILL BE OVERWRITTEN!!!
