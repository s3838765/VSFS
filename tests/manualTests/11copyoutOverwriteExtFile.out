This is the first line of an overwritten file.
This is the second.
This is the third and final line!
